source/Government.cpp

```diff
@@ -366,6 +374,12 @@ void Government::Load(const DataNode &node)
 			child.PrintTrace("Skipping unrecognized attribute:");
 	}
 
+	// Ensure reputation minimum is not above the
+	// maximum, and set reputation again to enforce limtis.
+	if(reputationMin > reputationMax)
+		reputationMin = reputationMax;
+	SetReputation(Reputation());
```

Comment:
Why?

Reply:
Right in the comment above. If the rep values for a player are somehow out of bounds when loading in, that'll set it back within the bounds.